src/tpm2.c

```diff
@@ -5284,7 +5286,9 @@ int TPM2_GPIO_Config(GpioConfig_In* in)
     }
     return rc;
 }
-#elif defined(WOLFTPM_NUVOTON)
+#endif /* WOLFTPM_ST33 */
+
+#ifdef WOLFTPM_NUVOTON
```

Comment:
Why change to exclusive logic? It seems like only one can be compile in at a time.